src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <label
+      class="form_box"
+      for="search_big"
+    >
+      <form
+        action="localhost"
+        method="post"
+        data-qa="big"
+      >
+        <input
+          data-qa="keypress"
+          class="search search_big"
+          type="text"
+          id="search_big"
+          placeholder="Try &quot;Los Angeles&quot"
+        >
+      </form>
+    </label>
```

Comment:
the HTML structure  is wrong, there should be` label` for `input` not for `form`
so it should be form - label - input